cmd/vela-email/plugin.go

```diff
@@ -118,6 +120,22 @@ func (p *Plugin) Validate() error {
 		return ErrorMissingEmailFromParam
 	}
 
+	if len(p.Attachment.Filename) != 0 {
+		fileInfo, err := os.Stat(p.Attachment.Filename)
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
are there other errors we should worry about?

Reply:
any other errors will get handled by the email library and returned